src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -355,17 +355,21 @@ describe('Step Registration', () => {
         const registrationRadio = screen.getByLabelText('Embed an activation key and register systems on first boot');
         userEvent.click(registrationRadio);
 
+        let organizationId;
+        await waitFor(() => {
+            organizationId = screen.getByLabelText('Organization ID');
+        });
+        expect(organizationId).toHaveValue('5');
+        expect(organizationId).toBeDisabled();
+
+        let activationKey;
         await waitFor(() => {
-            const organizationId = screen.getByLabelText('Organization ID');
-            expect(organizationId).toHaveValue('5');
-            expect(organizationId).toBeDisabled();
             // can't getByLabelText b/c the label contains an extra <span>
             // with a `*` to denote required
-            const activationKey = screen.getByTestId('subscription-activation');
-            expect(activationKey).toHaveValue('');
-            expect(activationKey).toBeEnabled();
-            // expect(activationKey).toBeRequired(); DDF does not support required fields
+            activationKey = screen.getByTestId('subscription-activation');
```

Comment:
Could replace this with await screen.findByTestId?

Reply:
Yep! Updated.